app/views/root/_publication.html.erb

```diff
@@ -10,7 +10,9 @@
       <% end %>
     </h4>
 
-    <% if publication.safe_to_preview? %>
+    <% if publication.published? %>
+      <%= link_to "#{Plek.find('www')}/#{publication.slug}", class: 'link-muted' %>
```

Comment:
Same as above.

It's better to use `Plek.new.website_root`, as this uses `GOVUK_WEBSITE_ROOT` environment variable if it is set.